docs/pages/includes/docker-images-oss.mdx

```diff
@@ -10,9 +10,17 @@ version of Teleport from the three most recent release branches. They are
 stable, and we recommend their use to keep your Teleport installation up to
 date.
 
-|Image name|Teleport version|Image automatically updated?|Image base|
+|Image name|Teleport version|Image automatically updated?|Troubleshooting Tools?|Image base|
```

Comment:
It looks like the table isn't rendering properly on the dev server (`yarn dev`):

<img width="857" alt="Screenshot 2023-04-04 at 6 04 03 PM" src="https://user-images.githubusercontent.com/25250668/229932917-143d0cc9-a963-47b9-ad17-bf97efadda5b.png">

Reply:
Fixed